Makefile

```diff
@@ -1,3 +1,12 @@
+# Set ENV to 'YELP' if FQDN ends in '.yelpcorp.com'
```

Comment:
nit: I don’t think this comment applies here 🙂

Reply:
You're right, I misread what it meant haha